src/index.html

```diff
@@ -8,100 +8,153 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
 
         <title>ESFiddle</title>
+        <script src="https://use.fontawesome.com/52284be701.js"></script>
```

Comment:
Same thing as the about page, could you move the js file to just before the closing body tag. Preferably include the minified version too please 👍